packages/icons/src/library/saved.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const saved = (
+	<SVG
+		aria-hidden
+		role="img"
+		focusable="false"
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="-2 -2 24 24"
```

Comment:
@jasmussen @senadir  Since we're just adding dashicons to this new package manually as we refactor the Gutenberg icons, I think instead of trying to fix the 20/24 bug programmatically, we should just apply this viewBox to the dashicons SVG as we add them to the package.